2022-04-28 16:03:19,782 INFO    MainThread:32337 [internal.py:wandb_internal():90] W&B internal server running at pid: 32337, started at: 2022-04-28 16:03:19.782081
2022-04-28 16:03:19,785 DEBUG   HandlerThread:32337 [handler.py:handle_request():141] handle_request: check_version
2022-04-28 16:03:19,786 INFO    WriterThread:32337 [datastore.py:open_for_write():75] open: /home/jmj3047/mj_MLP_prac/wandb/run-20220428_160318-262vrq5i/run-262vrq5i.wandb
2022-04-28 16:03:19,790 DEBUG   SenderThread:32337 [sender.py:send():232] send: header
2022-04-28 16:03:19,790 DEBUG   SenderThread:32337 [sender.py:send_request():246] send_request: check_version
2022-04-28 16:03:19,858 DEBUG   SenderThread:32337 [sender.py:send():232] send: run
2022-04-28 16:03:20,124 INFO    SenderThread:32337 [dir_watcher.py:__init__():166] watching files in: /home/jmj3047/mj_MLP_prac/wandb/run-20220428_160318-262vrq5i/files
2022-04-28 16:03:20,124 INFO    SenderThread:32337 [sender.py:_start_run_threads():811] run started: 262vrq5i with start time 1651129398
2022-04-28 16:03:20,124 DEBUG   SenderThread:32337 [sender.py:send():232] send: summary
2022-04-28 16:03:20,125 DEBUG   HandlerThread:32337 [handler.py:handle_request():141] handle_request: run_start
2022-04-28 16:03:20,126 INFO    SenderThread:32337 [sender.py:_save_file():946] saving file wandb-summary.json with policy end
2022-04-28 16:03:21,131 INFO    Thread-8  :32337 [dir_watcher.py:_on_file_created():213] file/dir created: /home/jmj3047/mj_MLP_prac/wandb/run-20220428_160318-262vrq5i/files/wandb-summary.json
2022-04-28 16:03:21,724 DEBUG   HandlerThread:32337 [meta.py:__init__():35] meta init
2022-04-28 16:03:21,724 DEBUG   HandlerThread:32337 [meta.py:__init__():49] meta init done
2022-04-28 16:03:21,724 DEBUG   HandlerThread:32337 [meta.py:probe():209] probe
2022-04-28 16:03:21,859 INFO    WriterThread:32337 [datastore.py:close():279] close: /home/jmj3047/mj_MLP_prac/wandb/run-20220428_160318-262vrq5i/run-262vrq5i.wandb
2022-04-28 16:03:22,131 INFO    SenderThread:32337 [sender.py:finish():1106] shutting down sender
2022-04-28 16:03:22,131 INFO    SenderThread:32337 [dir_watcher.py:finish():279] shutting down directory watcher
2022-04-28 16:03:22,133 INFO    SenderThread:32337 [dir_watcher.py:finish():309] scan: /home/jmj3047/mj_MLP_prac/wandb/run-20220428_160318-262vrq5i/files
2022-04-28 16:03:22,134 INFO    SenderThread:32337 [dir_watcher.py:finish():323] scan save: /home/jmj3047/mj_MLP_prac/wandb/run-20220428_160318-262vrq5i/files/config.yaml config.yaml
2022-04-28 16:03:22,135 INFO    SenderThread:32337 [dir_watcher.py:finish():323] scan save: /home/jmj3047/mj_MLP_prac/wandb/run-20220428_160318-262vrq5i/files/wandb-summary.json wandb-summary.json
2022-04-28 16:03:22,136 INFO    SenderThread:32337 [file_pusher.py:finish():145] shutting down file pusher
2022-04-28 16:03:22,136 INFO    SenderThread:32337 [file_pusher.py:join():150] waiting for file pusher
2022-04-28 16:03:22,841 INFO    Thread-11 :32337 [upload_job.py:push():137] Uploaded file /home/jmj3047/mj_MLP_prac/wandb/run-20220428_160318-262vrq5i/files/config.yaml
2022-04-28 16:03:22,885 INFO    Thread-12 :32337 [upload_job.py:push():137] Uploaded file /home/jmj3047/mj_MLP_prac/wandb/run-20220428_160318-262vrq5i/files/wandb-summary.json
2022-04-28 16:03:23,518 ERROR   MainThread:32337 [internal.py:wandb_internal():165] Thread HandlerThread:
Traceback (most recent call last):
  File "/home/jmj3047/anaconda3/lib/python3.8/site-packages/wandb/sdk/internal/internal_util.py", line 51, in run
    self._run()
  File "/home/jmj3047/anaconda3/lib/python3.8/site-packages/wandb/sdk/internal/internal_util.py", line 102, in _run
    self._process(record)
  File "/home/jmj3047/anaconda3/lib/python3.8/site-packages/wandb/sdk/internal/internal.py", line 265, in _process
    self._hm.handle(record)
  File "/home/jmj3047/anaconda3/lib/python3.8/site-packages/wandb/sdk/internal/handler.py", line 133, in handle
    handler(record)
  File "/home/jmj3047/anaconda3/lib/python3.8/site-packages/wandb/sdk/internal/handler.py", line 143, in handle_request
    handler(record)
  File "/home/jmj3047/anaconda3/lib/python3.8/site-packages/wandb/sdk/internal/handler.py", line 672, in handle_request_run_start
    run_meta.probe()
  File "/home/jmj3047/anaconda3/lib/python3.8/site-packages/wandb/sdk/internal/meta.py", line 230, in probe
    self._setup_git()
  File "/home/jmj3047/anaconda3/lib/python3.8/site-packages/wandb/sdk/internal/meta.py", line 197, in _setup_git
    if self._settings.disable_git or not self._git.enabled:
AttributeError: 'SettingsStatic' object has no attribute 'disable_git'
2022-04-28 16:03:25,533 INFO    MainThread:32337 [internal.py:handle_exit():80] Internal process exited
